include/wf.hrl

```diff
@@ -947,11 +948,6 @@
         function                :: atom() | text(),
         args=[]                 :: [text()]
     }).
--record(if_value, {?ACTION_BASE(action_if_value),
```

Comment:
Why did you remove this element?

Reply:
My wf.hrl was not the latest ('action_if_value' was added in April I see now).

Please ignore this, should still be there